### DISCUSSION TITLE
HA installation as an addon not working

### DISCUSSION THREAD

**Discussion by Wolfang42** [mrlt8/docker-wyze-bridge#1219] [Q&A] [answered]
Since the latest update, the Wyze Pan v3 cams I have are not working other than for stills.  I have installed via the HA integration. The cams are discovered but only stills are presented. In the WebUI of the Wyze Bridge, when I try to "start" the stream, I get the error saying lost connection with bridge.

Do I need to have a docker-compose.yaml file if I have installed from HA addon?

**Comment by mrlt8** [comment]
Can you post some logs?

**Reply by Wolfang42** [reply]
Which logs are best for this?  Reloading HA in the Docker Wyze Bridge web interface I can now  hit play buttons and get live streaming on the two cameras. But I am not able to find a way to successfully integrate them to get any picture in a dashboard card.

**Reply by mrlt8** [reply]
Sounds like an authentication issue. We enabled authentication on all streams starting with v2.9.0. 
The user name for the streams should be `wb` and the password can be found on the bottom of your web ui:

<img width="447" alt="random key" src="https://github.com/mrlt8/docker-wyze-bridge/assets/67088095/256f64e7-64e2-451a-8308-aeba66cb8e29">

you should be able to add the credentials to your stream with something like this:
```
rtsp://wb:myWBapiKey@127.0.0.1:8443/cam-name
```

If you're running the bridge entirely locally, you can disable authentication by setting `WB_AUTH` to false.

**Reply by Wolfang42** [reply]
This works for getting a correct address that is accessible with an RTSP player. Thank you! 

But what of adding to a dashboard card?  I add the entity but no picture shows up, only "unavailable."

Is there some new code I need in the configuration.yaml that is not in the GUI for Generic Camera? And is the same pw needed for the still image?

![image](https://github.com/mrlt8/docker-wyze-bridge/assets/720722/e4cbc4c5-67b7-4d5f-85c8-291dbc428bc9)

**Reply by Wolfang42** [reply]
Cannot seem to find the right snapshot address...the RTSP is working fine, but without assigning the correct snapshot address, the configuration in Generic Camera will not complete.

**Reply by mrlt8** [reply, marked as answer]
I believe you should be able to use the api key to authenticate for the image as well:

```
http://192.168.4.111:5000/img/foyer.jpg?api=myapikey
```

**Reply by Wolfang42** [reply]
This worked! Thank you so much. I read through all the Wiki and did not see that solution.
